spotifyMini/Models/Artist.swift

```diff
@@ -61,6 +62,7 @@ class Artist {
                 self.topTracks = result.value
                 var userInfo: [String:AnyObject]?
                 if let error = result.error as? SpotifyError {
+                    userInfo = [String:AnyObject]()
```

Comment:
seeing that this line is repeated over and over again, you should find a way to consolidate. Consider making an NSNotificationCenter or NSNotification extension in another PR